### PR TITLE
Import android-components quarantine

### PR DESCRIPTION
**Pull request by mozilla-pontoon**
Import android-components quarantine

### CODE REVIEW

File: mozilla-mobile/android-components/components/browser/toolbar2/src/main/res/values/strings.xml
Comment (Delphine) [inline mozilla-mobile/android-components/components/browser/toolbar2/src/main/res/values/strings.xml:5]: Localizers only see one string at a time, and the comment will only appear the first string.
Please add a comment describing the "Clear" string as well

**Review by Delphine — CHANGES_REQUESTED** [review]
@eeejay: I think you landed this string. Could you please add a localization comment to it?
thanks!

File: mozilla-mobile/android-components/components/browser/toolbar2/src/main/res/values/strings.xml
Comment (rocketsroger) [reply mozilla-mobile/android-components/components/browser/toolbar2/src/main/res/values/strings.xml:5]: Yes, we are creating another toolbar from copying the original toolbar code.  Can we ignore this for today? The history did not correctly link so I'm going to remove the toolbar2 folder soon.

Comment (Delphine) [reply mozilla-mobile/android-components/components/browser/toolbar2/src/main/res/values/strings.xml:5]: Thanks for the update
I can put these strings on hold for now but as soon as new a-c strings land, they will be part of this PR and so I will be blocked on exposing them.
Do you expect the removal of toolbar2 folder to happen soon?

Comment (rocketsroger) [reply mozilla-mobile/android-components/components/browser/toolbar2/src/main/res/values/strings.xml:5]: yes, I'm currently removing it today.  One question, if I'm copying the toolbar folder into a new "toolbar-legacy" folder, I should update the clear string?  I can do that for both toolbar and "toolbar-legacy" (not sure the name yet)

Comment (rocketsroger) [reply mozilla-mobile/android-components/components/browser/toolbar2/src/main/res/values/strings.xml:5]: At the end we're sticking with the current toolbar and toolbar2.  I've put up a fix for [this](https://phabricator.services.mozilla.com/D208425).  I'll land it as soon as I get a r+.

**Review by Delphine — APPROVED** [review]

**Review by Delphine — APPROVED** [review]

**Pull request merged**
